notebooks/proj_sys_eng/sitcom_integration/l3_system_integ/LVV-T2241-MTAOS_accumulation.ipynb

```diff
@@ -0,0 +1,897 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "script.log.info(f\"{test_case} - {test_exec} - CamHex ready\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d0dbcf1d",
+   "metadata": {},
+   "source": [
+    "***"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f88dc8c8",
+   "metadata": {},
+   "source": [
+    "## Get M2Hex ready \n",
+    "- Check config \n",
+    "- Make sure LUT is on, and has valid inputs; \n",
+    "- Make sure hex is at LUT position"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "24a0a98e-80fb-4936-8c75-a5d363e7b69e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await mtcs.set_state(\n",
+    "    state=salobj.State.ENABLED, \n",
+    "    components=[\"mthexapod_2\"],\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ba87515d-98e1-4d15-9070-179e6fd532f5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await vandv.hexapod.get_hexapod_configuration(mtcs.rem.mthexapod_2)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ea2a9043-29d0-4672-9352-84e354f93d92",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await vandv.hexapod.check_hexapod_lut(mtcs.rem.mthexapod_2)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "61ae9586-4d59-450e-9ccf-0f6d6cc22eb0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await mtcs.reset_m2_hexapod_position()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "31c36f75-710f-4f65-9f81-3e4d974777e0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await vandv.hexapod.print_hexapod_compensation_values(mtcs.rem.mthexapod_2)\n",
+    "await vandv.hexapod.print_hexapod_position(mtcs.rem.mthexapod_2)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a714b1d8-dcae-45f5-9df6-bc9e5be94f20",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "script.log.info(f\"{test_case} - {test_exec} - M2Hex ready\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "deb6d9b3",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "## Slew to a target and track.\n",
+    "\n",
+    "Choose a target such that the rotator stays within a couple of degrees of its initial position. (az = 120º, el = 60º, rotator type = PhysicalSky, rot=1.9º)\n",
+    "\n",
+    "This is because the CCW might not be running (MTmount in simulation mode)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b0d5c479",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "target = await mtcs.find_target(el=60, az=120, mag_limit=8)\n",
+    "print(target)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7eb81386",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await mtcs.slew_object(target, rot_type=RotType.PhysicalSky, rot=1.9)"
```

Comment:
Perhaps add here safe = False to move mount in small steps?

Reply:
This is a stray commit that comes from another test case. This change should not be here.